alpha/apps/kaltura/lib/kContextDataHelper.php

```diff
@@ -152,6 +162,38 @@ public function setMediaProtocol($protocol) {
 		 $this->mediaProtocol = $protocol;
 	}
 
+	/**
+	 * @return int
+	 */
+	public function getClipFrom()
+	{
+		return $this->clipFrom;
+	}
+
+	/**
+	 * @param $clipFrom
+	 */
+	public function setClipFrom($clipFrom)
+	{
+		$this->clipFrom = $clipFrom;
```

Comment:
lets add logic that user cannot ask for clipFrom that that is longer than clip to and the opposite